guardrails_test.py

```diff
@@ -0,0 +1,96 @@
+import logging
+import time
+import pytest
+
+from cassandra import InvalidRequest
+
+from dtest import Tester, create_ks
+from tools.assertions import assert_one
+
+since = pytest.mark.since
+logger = logging.getLogger(__name__)
+
+class BaseGuardrailsTester(Tester):
+
+    def prepare(self, rf=1, options=None, nodes=3, install_byteman=False, extra_jvm_args=None, **kwargs):
+        if options is None:
+            options = {}
+
+        if extra_jvm_args is None:
+            extra_jvm_args = []
+
+        cluster = self.cluster
+        cluster.populate([nodes, 0], install_byteman=install_byteman)
```

Comment:
what's that magic with `[nodes, 0]`?
do we want to set the snitch in this way?

Reply:
Sorry, I don't know - I'm not familiar with it.

Reply:
@Gerrrr Any idea?

Reply:
number of nodes per dc... maybe? This can be verified by increasing the number and observing the cluster status.

If this is the case, we can probably remove 0.

Reply:
The test is passing without the 0, so I'll remove it.